Mage.Sets/src/mage/cards/s/ShovingMatch.java

```diff
@@ -50,7 +50,7 @@ public ShovingMatch(UUID ownerId, CardSetInfo setInfo) {
         super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{2}{U}");
 
         // Until end of turn, all creatures gain "{tap}: Tap target creature."
-        Ability ability = new SimpleActivatedAbility(new TapTargetEffect(), new TapSourceCost());
+        Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, new TapTargetEffect(), new TapSourceCost());
```

Comment:
This wasn't necessary, I was using the constructor I added that defaults to battlefield because most activated abilities are on the battlefield anyway. It doesn't really matter, I just figured I'd mention it.

Reply:
Missed the constructor, good to know.